packs/bit-bucket/config.yaml

```diff
@@ -0,0 +1,4 @@
+Host: https://bitbucket.org/api/1.0/
```

Comment:
Again, same as in the other PR - it's unlikely that this URL will ever change so you can just use a constant in the module - no need for it to be a configuration ion.